src/main/resources/defaults/data/SkillParticleGeneration.json

```diff
@@ -539,6 +549,16 @@
 			}
 		]
 	},
+	{
+		"avatarId": 10000065,
+		"name": "Kuki Shinobu",
+		"amountList": [
+			{
+				"value": 4,
```

Comment:
From what I can so in Shinobu's `BinOutput/Ability/Temp/AvatarAbilities/ConfigAbility_Avatar_Shinobu.json`, this value should be `1`, not `4`.

Reply:
Okay, changed